src/main/java/models/User.java

```diff
@@ -61,7 +61,7 @@ public int getScore() {
         }
 
         // If the score is over the 21
-        // and there is on ace remove 11 and add 1
+        // and there is one ace, remove 11 and add 1
```

Comment:
Just a random thing I saw, decided it would be easier to just change it here rather than making you resubmit a pull request over some text.